properties/service_fabric_common.props

```diff
@@ -23,7 +23,7 @@
     <!-- Version for binaries, nuget packages generated from this repo. -->
     <!-- TODO: Versions numbers are changed here manually for now, Integrate this with GitVersion. -->
     <MajorVersion>5</MajorVersion>
-    <MinorVersion>0</MinorVersion>
+    <MinorVersion>1</MinorVersion>
     <BuildVersion>1</BuildVersion>
```

Comment:
If we bump the minor version, build version should go back to 0. So, it should be `5.1.0`.